packages/rest/src/router/metadata.ts

```diff
@@ -166,14 +224,20 @@ function resolveControllerSpec(
  * @param constructor Controller class
  */
 export function getControllerSpec(constructor: Function): ControllerSpec {
-  let spec = Reflector.getOwnMetadata(API_SPEC_KEY, constructor);
+  const scope = Reflector.getOwnMetadata(CONTROLLER_SCOPES_KEY, constructor);
```

Comment:
Considering we allow multiple servers for a given controller, we can have the following:

```
export function getControllerSpec(constructor: Function, server?: string): ControllerSpec {
  // Build controller spec for the given server and cache it 
}
```